### DISCUSSION TITLE
Release v1.0.2 / 5.55.2 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1422] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes bugs introduced recently

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### Fixed
- fixed recovery window not refreshing count on reload [#1402](https://github.com/sandboxie-plus/Sandboxie/issues/1402)
- fixed printing issue introduced in 1.0.0 [#1397](https://github.com/sandboxie-plus/Sandboxie/issues/1397)
- fixed issues with CreateProcess function [#1408](https://github.com/sandboxie-plus/Sandboxie/issues/1408)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/'>Release v1.0.2 / 5.55.2</a>.</em>

**Comment by BilleBarrett** [comment]
Tried the fix on one client that uses AOL email and it didn't seem to work
I will contact the client in the morning, the other client's printer was offline
I did try on another client's computer monitoring the Print spooler and it work, saw the job get spooled and then cleared out.

**Comment by DavidXanatos** [comment]
According to my tests with a networked printer the issue was that due to a change for the rule processing feature in some situations a open file path directive was ignored, resulting in the print failing. this has been fixed now so imho it should work as before.

**Reply by BilleBarrett** [reply]
Got an email from one of the clients that do AOL prints, updated him last night "OK"
He Said, "It works just fine, thanks for the update!!!"
Thanks very much, now I can update the rest of my Sandboxie Clients
